File: vid-app-common/src/main/java/org/onap/vid/aai/model/OwningEntity.java
package org.onap.vid.aai.model;


import com.fasterxml.jackson.annotation.JsonAlias;

/**
 * Created by moriya1 on 08/10/2017.
 */
public class OwningEntity extends AaiRelationResponse {
    private String owningEntityId;
    private String owningEntityName;

    public String getOwningEntityId() {
        return owningEntityId;
    }

    @JsonAlias("owning-entity-id")
    public void setOwningEntityId(String owningEntityId) {
        this.owningEntityId = owningEntityId;
    }

    public String getOwningEntityName() {
        return owningEntityName;
    }

    @JsonAlias("owning-entity-name")
    public void setOwningEntityName(String owningEntityName) {
        this.owningEntityName = owningEntityName;
    }


}

File: vid-webpack-master/src/app/shared/models/owningEntity.ts
interface OwningEntityResponse {
  id: string,
  name: string
}

export class OwningEntity {
  id: string;
  name: string;

  constructor(serviceJson: OwningEntityResponse){
    this.id = serviceJson.id;
    this.name = serviceJson.name;
  }
}
